services/mlmodel/tflite_cpu/tflite_cpu_test.go

```diff
@@ -0,0 +1,113 @@
+package tflitecpu
+
+import (
+	"context"
+	"testing"
+
+	"github.com/nfnt/resize"
+	"go.viam.com/test"
+	"go.viam.com/utils/artifact"
+
+	"go.viam.com/rdk/rimage"
+	"go.viam.com/rdk/services/mlmodel"
+)
+
+func TestNewTFLiteCPUModel(t *testing.T) {
```

Comment:
Add a test that also uses a TFLite text classifier model, and see if you get the expected outputs. This model should handle more than just image inputs.

Reply:
Added!